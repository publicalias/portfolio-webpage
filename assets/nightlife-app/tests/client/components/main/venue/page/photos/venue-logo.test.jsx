"use strict";

//local imports

const VenueLogo = require("../../../../../../../scripts/client/components/main/venue/page/photos/venue-logo");

const { testWrapper } = require("../../../../../test-helpers");

//global imports

const { initTestSnapshot } = require("redux/tests/client-tests");
const { reactTests } = require("redux/tests/react-tests");

//utilities

const { testShallow } = testWrapper(VenueLogo);

const testSnapshot = initTestSnapshot(testShallow);

//setup

beforeAll(reactTests.setup);
beforeEach(reactTests.inject(VenueLogo));

//venue logo

test("VenueLogo should match snapshot", () => testSnapshot());
