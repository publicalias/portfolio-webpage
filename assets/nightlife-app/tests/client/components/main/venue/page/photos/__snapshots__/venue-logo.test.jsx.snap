// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`venue logo should match snapshot 1`] = `
<a
  href="https://www.yelp.com/"
>
  <img
    alt="Yelp Logo"
    className="c-venue-photos__logo"
    src="/nightlife-app/media/yelp-logo.png"
  />
</a>
`;
